Host authenticity keys (#3955)
circleci/circleci-docs: jekyll/_cci2/configuration-reference.md

- run: git submodule update --init
```

> This command will automatically add the required authenticity keys for interacting with GitHub and Bitbucket over SSH, which is detailed further here: [Authenticity of a host can't be established]({{ site.baseurl }}/2.0/gh-bb-integration/#authenticity-of-a-host-cant-be-established). This is helpful for implementing a custom checkout command as well.

Reviewer: ```suggestion
This command will automatically add the required authenticity keys for interacting with GitHub and Bitbucket over SSH, which is detailed further in our [integration guide]({{ site.baseurl }}/2.0/gh-bb-integration/#authenticity-of-a-host-cant-be-established) – this guide will also be helpful if you wish to implement a custom checkout command.
```
